lib/help.js

```diff
@@ -19,7 +19,12 @@ module.exports = (commandName, packageLink) => {
     {cyan --expiring, -e}    outputs detailed report of expiring packages only
     {cyan --supported, -s}   outputs detailed report of supported packages only
     {cyan --csv}             outputs csv file in the project path
+    {cyan --token, -t}       token to access raw file from github private instance
+    {cyan --hostUrl}         URL endpoint that returns package.json, yarn.lock and npmrc files.
```

Comment:
What does the payload of this endpoint need to look like? (maybe to be documented in the readme?)

Reply:
Sure. It must be something reachable. It must be a valid URL

Reply:
Added documentation in readme file.